contrib/core/actions/generate_uuid.py

```diff
@@ -0,0 +1,27 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import uuid
+from st2actions.runners.pythonrunner import Action
+
+
+class GenerateUUID(Action):
+    def run(self, uuid_type):
+        if uuid_type == 'uuid1':
+            return str(uuid.uuid1())
+        elif uuid_type == 'uuid4':
+            return str(uuid.uuid4())
+        else:
+            return False, "Unknown uuid_type. Only uuid1 and uuid4 are supported"
```

Comment:
Would be better and more Pythonic to throw an exception here - `raise ValueError('Unknown uuid_type. Only uuid1 and uuid4 are supported')`

Returning a tuple with a status code is mostly meant to be used when you want action to fail, but also want to return some structured data (e.g. dict, array, etc.).

If you just want to return error message, exceptions are usually a better fit.